src/commands/talk.test.ts

```diff
@@ -0,0 +1,116 @@
+// External dependencies
+import { ChannelType } from 'discord.js';
+
+// Overwrite dectalk say method
+const mockSay = jest.fn();
+const dectalk = jest.requireActual<typeof import('dectalk')>('dectalk');
+jest.mock('dectalk', () => ({
+	...dectalk,
+	say: mockSay,
+}));
+
+// Mock the logger so nothing is printed
+jest.mock('../logger');
+
+// Import the code to test
+import { talk } from './talk';
+
+describe('Talk Slash Command', () => {
+	const message = 'test';
+	let context: TextInputCommandContext;
+	const emptyBuffer: Buffer = Buffer.from([]);
+	const mockPrepare = jest.fn();
+	const mockReply = jest.fn();
+
+	beforeEach(() => {
+		context = {
+			interaction: {
+				options: {
+					data: {
+						length: 1,
+					},
+					getString(name: string, required: boolean) {
+						name;
+						required;
+						return message;
+					},
+					getInteger(name: number) {
+						name;
+						return undefined;
+					},
```

Comment:
When no parameter is used, and the function's type is defined elsewhere, we don't need to list the parameters. This means we don't need useless references either:

```suggestion
					getString: () => message,
					getInteger: () => undefined,
```